virtcontainers/hypervisor.go

```diff
@@ -104,6 +104,10 @@ const (
 
 	// memoryDevice is memory device type
 	memoryDev
+
+	// hybridVSockDev is a hybrid virtio-vsock device supported
+	// only on certain hypervisors, like firecracker.
+	hybridVSockDev
```

Comment:
How about renaming this and below names `hybridVirtioVsockDev` to be more explicit?

Reply:
ok, sounds good